ClientApp/package.json

```diff
@@ -3,10 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/react-fontawesome": "^0.1.16",
+    "@microsoft/signalr": "^5.0.11",
     "axios": "^0.22.0",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^5.1.3",
```

Comment:
We are using react-strap library (at least I did) and it currently supports only bootstrap 4. So at least for me, there were some issues like "username not found" always showing on the login popup. Version 4.6.0 seems to work better for me.

Reply:
If you don't see changes try: "npm prune" and "npm install" commands.

Reply:
Reverted back to the old version, all should be working fine still.